capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.collegescorecarddata;
```

Comment:
Note that the package name is correct (all lowercase), but the path of this interface is incorrect (`.../CollegeScoreCardData/...`).

Also, I suspect you'll not want your API interfaces in the `CollegeScoreCardData` package - since they shouldn't be inherently tied to CollegeScoreCard.

Reply:
moved it! Thanks!